extensions/wikia/ContributionPrototype/CPArticleRenderer.php

```diff
@@ -71,6 +71,7 @@ private function addStyles(OutputPage $output) {
 	private function addScripts(OutputPage $output) {
 		$output->addHTML("<script src=\"{$this->publicHost}/public/assets/vendor.dll.js\"></script>");
 		$output->addHTML("<script src=\"{$this->publicHost}/public/assets/app.js\"></script>");
+		$output->addHTML('<link href="https://opensource.keycdn.com/fontawesome/4.7.0/font-awesome.min.css" rel="stylesheet">');
```

Comment:
An interesting note here is that the fonts *do* load properly when using a production build. They don't load with a dev build because everything goes through the dev node server, which doesn't know how to serve those files. The consequence of this is that we'll load the fonts twice in prod, but that's probably OK.